### PR TITLE
Add support to scroll to anchors in the catalog

### PR DESCRIPTION
**Pull request by rtorrero**
This PR addresses https://github.com/trento-project/trento/issues/532 by allowing navigation from the check results modal to the checks catalog by clicking on the check description.

### CODE REVIEW

File: web/frontend/javascripts/catalog_scroll.js [hunk 1]
```diff
@@ -0,0 +1,10 @@
+const anchor = window.location.hash.slice(1);
+if (anchor) {
+  const anchorEl = '#' + anchor;
+  $(anchorEl).closest('.card').find('.collapse-toggle').click();
+  const checkElem = $(anchorEl).closest('tr').find('a');
+  checkElem.click();
+  setTimeout(() => {
+    checkElem[0].scrollIntoView({ behavior: 'smooth', block: 'end' });
```
Comment (fabriziosestito) [inline web/frontend/javascripts/catalog_scroll.js:8]: _smooth_

**Review by dottorblaster — APPROVED** [review]
YAY

**Comment by rtorrero** [comment]
@dottorblaster thanks for the support on this one!

**Comment by dottorblaster** [comment]
No problem mate :rocket: :sunglasses:

**Pull request merged**